cedar-wasm/src/policies_and_templates.rs

```diff
@@ -0,0 +1,250 @@
+use std::str::FromStr;
+
+use cedar_policy::{Policy, PolicySet};
+use serde::{Deserialize, Serialize};
+
+use tsify::Tsify;
+use wasm_bindgen::prelude::*;
+
+#[derive(Tsify, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+#[tsify(into_wasm_abi, from_wasm_abi)]
+pub enum JsonToPolicyResult {
+    Success { policy_text: String },
+    Error { errors: Vec<String> },
+}
+
+#[wasm_bindgen(js_name = "policyTextFromJson")]
+pub fn policy_text_from_json(json_str: &str) -> JsonToPolicyResult {
+    let parsed_json = match serde_json::from_str(json_str) {
+        Ok(v) => v,
+        Err(e) => {
+            return JsonToPolicyResult::Error {
+                errors: vec![e.to_string()],
+            }
+        }
+    };
+    let policy = Policy::from_json(None, parsed_json);
```

Comment:
This provides no way to users to set the policy id, so the ids will all overlap, unless you edit it later

Reply:
i don't think there's a use case for policy id's in this stateless context

Reply:
Yes, this will set the id to `policy0`. I think that's fine, since this function returns the policy source text (which has no way to represent ids).